Content.Server/Antag/Components/AntagSelectionComponent.cs

```diff
@@ -26,22 +32,27 @@ public sealed partial class AntagSelectionComponent : Component
     public List<AntagSelectionDefinition> Definitions = new();
 
     /// <summary>
-    /// The minds and original names of the players selected to be antagonists.
+    /// The minds and original names of the players assigned to be antagonists.
     /// </summary>
     [DataField]
-    public List<(EntityUid, string)> SelectedMinds = new();
+    public List<(EntityUid, string)> AssignedMinds = new();
 
     /// <summary>
     /// When the antag selection will occur.
     /// </summary>
     [DataField]
     public AntagSelectionTime SelectionTime = AntagSelectionTime.PostPlayerSpawn;
 
+    /// <summary>
+    /// Cached sessions of players who are chosen yet not given the role yet.
+    /// </summary>
+    public HashSet<ICommonSession> PreSelectedSessions = new();
+
     /// <summary>
     /// Cached sessions of players who are chosen. Used so we don't have to rebuild the pool multiple times in a tick.
     /// Is not serialized.
     /// </summary>
-    public HashSet<ICommonSession> SelectedSessions = new();
+    public HashSet<ICommonSession> AssignedSessions = new();
 
```

Comment:
Not much point me reviewing this as it's over my head but a lot of the line changes are just coping with this "selected" -> "assigned" name shift. You can reduce line changes by not doing this. This isn't that bad given this is restricted to AntagSelectionSystem but could still be annoying for a downstream to deal with.

Reply:
The `Selected` is no longer a good variable name for what the HashSet/bool is, as it reads ambiguous with the changes to how antags are first pre-selected and then assigned. `Assigned` is a more appropriate due to this.